interfaces/cython/cantera/test/test_composite.py

```diff
@@ -145,6 +145,33 @@ def test_collect_data(self):
         self.assertIn('X', collected)
         self.assertEqual(collected['X'].shape, (0, self.gas.n_species))
 
+    def test_append_no_norm_data(self):
+        gas = ct.Solution('h2o2.yaml')
+        gas.TP = 300, ct.one_atm
+        gas.X = gas.X - 1.e-16
+        states = ct.SolutionArray(gas)
+        states.append(T=gas.T, P=gas.P, X = gas.X, normalize = False)
+        self.assertEqual(states[0].T, gas.T)
+        self.assertEqual(states[0].P, gas.P)
+        self.assertArrayNear(states[0].X, gas.X)
+
+    def test_import_no_norm_data(self):
+        outfile = pjoin(self.test_work_dir, 'solutionarray.h5')
+        if os.path.exists(outfile):
+            os.remove(outfile)
+
+        gas = ct.Solution('h2o2.yaml')
```

Comment:
Please use double quotes in all new Python code. Thank you 😄

Reply:
@bryanwweber: On this note, it may make sense to update `CONTRIBUTING.md` ... there are also references to Python versions that are no longer relevant there. As I believe that you have a better overview of everything Python, I won't create a PR myself.